conformance/tests/typeddicts_extra_items.py

```diff
@@ -178,15 +181,18 @@ class Child(Parent, extra_items=int): # E: Cannot change 'extra_items' type unle
 class MovieBase2(TypedDict, extra_items=int | None):
     name: str
 
-class MovieRequiredYear(MovieBase2):  # E: Required key 'year' is not known to 'MovieBase'
-    year: int | None
+class MovieRequiredYear(MovieBase2):  # E[MovieRequiredYear]: Required key 'year' is not known to 'MovieBase'
+    year: int | None  # E[MovieRequiredYear]
 
-class MovieNotRequiredYear(MovieBase2):  # E: 'int | None' is not consistent with 'int'
-    year: NotRequired[int]
+class MovieNotRequiredYear(MovieBase2):  # E[MovieNotRequiredYear]: 'int | None' is not consistent with 'int'
+    year: NotRequired[int]  # E[MovieNotRequiredYear]
 
 class MovieWithYear(MovieBase):  # OK
     year: NotRequired[int | None]
 
+class MovieWithDirector(MovieBase):  # E[MovieWithDirector]: 'str' is not assignable to 'int | None'
```

Comment:
Maybe use MovieBase2? MovieBase was defined a long way up (line 33) and has ReadOnly extra items, which adds an additional complication. The test immediately above this one probably should also have used MovieBase2.

Reply:
I intentionally used `MovieBase` in order to test `ReadOnly` extra items; the non-`ReadOnly` variant is already covered. Do you think it would be better to define a `MovieBase3` instead of reusing `MovieBase`?

Reply:
I think so, it's hard to follow the tests when we reuse something that is defined so far up.

We could also use something other than movies in the test (heresy?).

Reply:
Indeed heresy ;) Done.